src/localization/localize.js

```diff
@@ -12,56 +12,54 @@
 * See the License for the specific language governing permissions and
 * limitations under the License.
 **/
-import LocalizedStrings from 'react-localization';
 import React from 'react';
 
+import * as en_bundle from './bundles/en.json';
+import * as en_bundle_branding from './bundles/en_branding.json';
+import * as ja_bundle from './bundles/ja.json';
+import * as ja_bundle_branding from './bundles/ja_branding.json';
 
-var supportedLangs = ['en', 'ja'];
-var translationData, bundlename, brandingData, bundlebrandingname, allTranslatedData, catalog = {};
+export const en = 'en';
+export const ja = 'ja';
 
-for (var i = 0; i < supportedLangs.length; i++) {
-  bundlename = './bundles/' + supportedLangs[i] + '.json';
-  //require doesn't interpret this as a string correctly unless its converted
-  //easiest conversion is to use + ''
-  translationData = require(bundlename + '');
-
-  // for branding
-  bundlebrandingname = './bundles/' + supportedLangs[i] + '_branding.json';
-  brandingData = require(bundlebrandingname + '');
-
-  // combine branding bundle
-  allTranslatedData = Object.assign(translationData, brandingData);
-  catalog[supportedLangs[i]] = allTranslatedData;
-}
+const bundles = {
+  [en]: {
+    ...en_bundle.default,
+    ...en_bundle_branding.default,
+  },
+  [ja]: {
+    ...ja_bundle.default,
+    ...ja_bundle_branding.default,
+  }
+};
 
-// window.navigator.language is the language that user sets at the top of
-// installed and enabled languages like en-US, en, ja-JP, ja, zh-CN, zh
-// since we have language without country like en or ja as supported languages,
-// need to find an exact match or a match of the language part
-var strings = new LocalizedStrings(catalog);
+const supportedLangs = [en, ja];
 
-var findLang =
+var foundLang =
   supportedLangs.find(sLang => window.navigator.language === sLang) ||
   supportedLangs.find(sLang => window.navigator.language.substring(0, 2) === sLang.substring(0, 2));
 
-if(findLang) {
-  strings.setLanguage(findLang);
-}
-else { //default
```

Comment:
the removal of a default language means that anyone who uses a language other than japanese or english ends up with non-translated content (even if they have English or Japanese later in their languages list)